From 9c949432554129ec42895aa1f2ab497a0ed91b39 Mon Sep 17 00:00:00 2001
From: Mathias Gibbens <gibmat@debian.org>
Date: Sat, 27 Jan 2024 16:11:20 +0000
Subject: [PATCH 02/11] Fix alpine template

On a Debian sid system with lxc 5.0.3 and the latest snapshot of lxc-templates,
attempting to create an alpine container fails with the error "mknod:
dev/console: File exists". This had been reported a while back in Debian
bug #998095 and partially addressed in #46.

I performed some testing of the resulting alpine container, and things appear
to be working correctly.

Signed-off-by: Mathias Gibbens <gibmat@debian.org>
---
 templates/lxc-alpine.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/templates/lxc-alpine.in b/templates/lxc-alpine.in
index 18f163c..535d0ef 100644
--- a/templates/lxc-alpine.in
+++ b/templates/lxc-alpine.in
@@ -294,7 +294,6 @@ make_dev_nodes() {
 
 	mknod -m 666 dev/tty c 5 0
 	chown 0:5 dev/tty  # root:tty
-	mknod -m 620 dev/console c 5 1
 	mknod -m 666 dev/ptmx c 5 2
 	chown 0:5 dev/ptmx  # root:tty
 }
-- 
2.47.1

